TheWitcher.Business/Services/QuestService.cs

```diff
@@ -24,7 +27,11 @@ public List<QuestDTO> GetNameIdLeadTimeQuests(HeroesDTO hero)
             var listQuestDTO = new List<QuestDTO>();
             if (hero != null)
             {
-                List<Quest> quests = _questRepository.GetItemList().Where(x => x.AccessLevel >= hero.HeroLevel).ToList();
+                IEnumerable<Quest> quests = _questRepository.GetItemList()?.Where(x => x.AccessLevel >= hero.HeroLevel).ToList();
```

Comment:
Let's change it. Inside repo return new empty List instead of null. Then, you will not have to check it for null